src/main/java/com/google/codeu/data/Message.java

```diff
@@ -18,14 +18,28 @@
 
 import java.util.UUID;
 
-/** A single message posted by a user. */
+/** The sting recipient helps to display messages received from another user. */
```

Comment:
Good job adding comments! However, this comment is describing the entire class, which is still a message posted by a user, not just the new string you've added. Maybe change this back, or modify it to "A single message posted from a user to a recipient"?

Reply:
Thank you so much! I will implement the change

Reply:
Awesome! Add more commits to your same branch and then run `git push` again so we can see them here. 👍